recipes/micropip/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "micropip" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/micropip-{{ version }}.tar.gz
+  sha256: 46e901e045860443808b9fd5ec7ad2ae7253ba25440c972b6d1d39e8b3f56f8a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
```

Comment:
Thanks for doing this @yuvipanda!

Could you please add the following runtime requirements,
https://github.com/pyodide/pyodide/blob/main/packages/micropip/meta.yaml#L16

Looks like there were missing https://github.com/pyodide/micropip/issues/21 but we also didn't have many users of this package outside of Pyodide so far :)

Reply:
And also https://pypi.org/project/pyodide-py/ is another dependency, you would likely need to package it as well (it's just another pure Python package).

Finally, Python 3.7+ is misleading, it actually needs 3.10+ . We could support a broader range of versions if there is demand for it.